Ordering/Contracts/OrderManagement/OrderItem.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Contracts
+{
+    public class OrderItem : IOrderItem
+    {
+        public static int CurrentItemID = 0;
+
+        public int ItemID { get; }
+        public string Name { get; set; }
+        public string ItemDescription { get; set; }
+        public double Price { get; set; }
+        public DateTime DateLastOrdered { get; set; }
+        public OrderItemCategory Category { get; set; }
+
+        public OrderItem(string name, string itemDescription, double price, DateTime date, OrderItemCategory cateogry)
+        {
+            ItemID = CurrentItemID;
```

Comment:
What will the ItemID represent, are you going to generate this ID inside this class?
If so, please explain based on what you will be generating it.
I suggest you discuss this with Jordan and consider where you will be needing this ID and write down the scenario to ensure this model works.

Reply:
I have declared a static variable called currentItemID. When an orderitem is initialized, we will assign orderitem's id with the currentItemID value and the currentItemID will increment by one. OrderItem's ID will always be unique when someone initializes orderItem variable.